doc/error-handling.md

```diff
@@ -136,17 +137,17 @@ Error logging shall be done by the logger only, no calls to std::cerr or similar
 All the methods presented (cxx::expected, Expects and Ensures and the error handler) can be used in posh. The appropriate way depends on the type of error scenario (cf. the respective sections for examples). The error handler should be considered the last option.
 
 ## Error Handling in utils
-Error logging is currently done by calls to cerr. In the future those might be redirected to the logger.
+Error logging is currently done by calls to std::cerr. In the future those might be redirected to the logger.
 
 The error handler cannot be used in utils. 
 
 Whether it is appropriate to use cxx::expected even if STL compatibility is broken by doing so depends on the circumstances and needs to be decided on a case-by-case basis. If the function has no STL counterpart cxx::expected can be used freely to communicate potential failure to the caller.
 
-It should be noted that since currently Expects and Ensures are active at release mode, prolific usage of these will incur a runtime cost. Since this is likely to change in the future, it is still advised to use them to document the developers intentions.
+It should be noted that since currently Expects and Ensures are active at Release Mode, prolific usage of these will incur a runtime cost. Since this is likely to change in the future, it is still advised to use them to document the developers intentions.
```

Comment:
Why did you capitalize "Release Mode" and "Debug Mode" ?
I would use bold or italics for emphasis... looks kind of weird to me doing it like this.

Reply:
That being said, I don't feel strongly about it. Feel free to ignore.

Reply:
Because @FerdinandSpitzschnueffler  likes it like this I guess... ;-) I am also impartial, I can change it to release mode in italics. Bold is too...bold for me at this occassion. :-)

Reply:
Let's just leave it haha.

Reply:
Actually I don't care how it is written, but I like consistency ;)